code/obj/item/clothing/masks.dm

```diff
@@ -115,6 +115,25 @@
 		setProperty("heatprot", 7)
 		setProperty("disorient_resist_eye", 10)
 
+/obj/item/clothing/mask/gasNT
```

Comment:
why not make this a child of gas mask?

Reply:
I had a reason for this last night when I made it but damned if I can think of why now. Good idea I'll fix that now.